packages/calcite-components-react/README.md

```diff
@@ -1,22 +1,22 @@
 # Calcite Components React
 
-A set of React components that wrap [Calcite Components](https://developers.arcgis.com/calcite-design-system/components/). An application using this package is provided in the [`calcite-components-examples`](https://github.com/Esri/calcite-components-examples) repo.
+This package provides React wrappers for [Calcite components](https://developers.arcgis.com/calcite-design-system/components/). See the [react example](https://github.com/Esri/calcite-design-system/tree/dev/examples/components/react) for a minimal application using this package.
 
 ## Installation
 
 ```sh
 npm install --save @esri/calcite-components-react
 ```
 
-This package includes the compatible version of the main component library as a dependency, so no need to install `@esri/calcite-components` separately.
+This package includes the compatible version of the standard `@esri/calcite-components` package, so you do not need to install it separately.
 
 ## Choose a build
 
-There are two builds that are provided by the standard `calcite-components` package.
+There are two builds provided by the standard components package.
 
 ### Custom Elements build
 
-[Custom Elements](developers.arcgis.com/calcite-design-system/get-started#custom-elements) is the recommended build when using frontend frameworks, such as React. To use this build, you will need to set the path to the `calcite-components` assets. You can either use local assets, which will be explained in a subsequent step, or assets hosted on a CDN.
+[Custom Elements](developers.arcgis.com/calcite-design-system/get-started#custom-elements) is the recommended build when using frontend frameworks, such as React. To use this build, you will need to set the path to the components assets. You can either use local assets, which will be explained in a subsequent step, or assets hosted on a CDN.
 
 ```jsx
 import { setAssetPath } from "@esri/calcite-components/dist/components";
```

Comment:
Would the commented out `setAssetPath` need to removed, or edited to reflect more of a local/different cdn, maybe similar to our [doc site example](https://developers.arcgis.com/calcite-design-system/get-started/#custom-elements-build):

```
setAssetPath("/path-to-your-assets/");
```

Reply:
I think it's fine as it is because it's where React apps commonly serve their assets, e.g. our React example:

https://github.com/Esri/calcite-design-system/blob/4e951e744b24f9ce4cde32740b20836b72a6c448/examples/components/react/src/main.tsx#L8

It's generalized in the doc because it's not the same for all frameworks or build tools, e.g. our rollup example:

https://github.com/Esri/calcite-design-system/blob/4e951e744b24f9ce4cde32740b20836b72a6c448/examples/components/rollup/src/main.js#L7

Reply:
I guess we could remove the CDN comment though because I think Lumina does that be default now

Reply:
@maxpatiiuk is that correct? And is there a new default behavior for `defineCustomElements`?

Reply:
> I guess we could remove the commented snippet for CDN assets though because I think Lumina uses the CDN by default when setAssetPath isn't called.

Correct. in map-components 4.32 we only document setAssetPath for people who don't want to use our CDN/need offline deployment

Reply:
Yes, defineCustomElements has the same behaviour
Also, if resourceUrl is provided to defineCustomElements, it simply calls setAssetPath for you - equivalent to calling setAssetPath directly.

Reply:
The path can be simplified:

```suggestion
import { setAssetPath } from "@esri/calcite-components";
```